src/objects/oo/zcl_abapgit_oo_class.clas.abap

```diff
@@ -968,4 +970,31 @@ CLASS zcl_abapgit_oo_class IMPLEMENTATION.
     lv_cp = cl_oo_classname_service=>get_classpool_name( iv_class_name ).
     READ TEXTPOOL lv_cp INTO rt_text_pool LANGUAGE iv_language. "#EC CI_READ_REP
   ENDMETHOD.
+
+
+  METHOD zif_abapgit_oo_object_fnc~syntax_check.
+    DATA:
+      ls_clskey      TYPE seoclskey,
+      lv_syntaxerror TYPE seox_boolean.
```

Comment:
can `abap_bool` be used instead of `seox_boolean` ?

Reply:
Sure. Done.